omod/src/main/resources/config.xml

```diff
@@ -177,6 +177,12 @@
         <defaultValue>7b0f5697-27e3-40c4-8bae-f4049abfb4ed</defaultValue>
     </globalProperty>
 
+    <globalProperty>
+		<property>ugandaemr.maximumPatientsPerDay</property>
+        <description>Number of patients that can be seen in a day</description>
```

Comment:
@reagan-meant Change description to "The maxiumum number of patients that can have appointments on a single day"

Reply:
@ssmusoke Okay